pkg/logs/input/journald/tailer_test.go

```diff
@@ -102,7 +102,7 @@ func TestContent(t *testing.T) {
 		&sdjournal.JournalEntry{
 			Fields: map[string]string{
 				sdjournal.SD_JOURNAL_FIELD_MESSAGE: "bar",
-				"_A":                               "foo.service",
+				"_A": "foo.service",
```

Comment:
This seems to be breaking your tests, I had the same problem, sometimes my IDE or `gofmt` reformat it like this too. Let me know if you find a solution to this